### ISSUE TITLE
Decouple upcoming queue display from board width

### ISSUE THREAD

**Issue by thunderrabbit** [thunderrabbit/robdot_conswi#16]
Currently, Helpers displays the upcoming pieces queue in the board{}, but this breaks if the board is not as wide as the queue is long.

For level 1, I would prefer the board to be width 1, so the user cannot possibly lose, but (as it is currently written,) the queue requires width 5 as a minimum.

I would probably make the queue part of a HUD, floating over the game board:

1. Make a HUD scene in subscenes/
2. Create script HUD.gd for HUD scene
3. Add `func display_queue(queue)` to HUD.gd
4. Make sure it shows up above the board.  Maybe make the pieces be transparent

**Comment by nooj** [comment]
I disagree with this.  It's not a priority right now, and only exists because of your desire to have level 1 be a weird shape.  Change this desire for level 1 and the issue goes away (at least for a while).

Also, level 1 shouldn't have width 1 because it will be more confusing to have a weird level shape than it will to learn to play the game on a typical level shape.

Again, this kind of issue isn't a priority right now.  Let's work around it until we have a playable game.  Tutorials and fun GUI stuff can wait will later.

**Comment by nooj** [comment]
Ok, instead of closing the issue, I'm adding a tag that will remind us not to mess with it till later.
